model/config.go

```diff
@@ -109,19 +109,19 @@ func (c *Config) runWorker(instanceName string, stop chan struct{}, done *sync.W
 	}
 }
 
-func (c *Config) runTest(tracer trace.Tracer) {
+func (c *Config) runTest(tracerProvider trace.TracerProvider) {
 	rootSvc := c.Services[0]
 	inst := rootSvc.instances[0]
 	endpoint := inst.Endpoints[0]
-
+	tracer := tracerProvider.Tracer("otel") // TODO: Need to verify this line of code
```

Comment:
the argument is meant to describe the piece of code that is being instrumented.
```suggestion
	tracer := tracerProvider.Tracer("worker")
```